scripts/src/main/resources/scripts/command/az

```diff
@@ -9,10 +9,11 @@ then
   fi
   exit
 fi
-if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
-TOOL_VERSION_COMMAND="az -v"
 # shellcheck source=scripts/functions
 source "$(dirname "${0}")"/../functions
+TOOL_VERSION_COMMAND="az -v"
+# shellcheck source=scripts/commandlet-cli
+source "$(dirname "${0}")"/../commandlet-cli
 
```

Comment:
Firstly, it may be that someome recently mentioned this problem(?)
When trying to setup azure on linux mint the setup of python is needed. 
It seems as if the python download url changed slightly, but significantly.
It turned out, that the version in the download-url isn't 3.12.0, as our script would expect, but 3.12.0a1. 
Expected download-url: https://www.python.org/ftp/python/3.12.0/Python-3.12.0.tgz .
Actual download-url: https://www.python.org/ftp/python/3.12.0/Python-3.12.0a1.tgz .
Therefore the download just fails. This is another argument for an manually editable list of versions I guess.